.env

```diff
@@ -0,0 +1 @@
+REACT_APP_LOCATION_IQ_KEY=pk.5b978d105e102fbfdc70c18da1625112
```

Comment:
HI there!

Feedback here is to get .env properly .gitignored. Add `.env` to `.gitignore`. You may find that you have to first remove env, then add it to gitignore then commit. Then add env variables after etc.

Reply:
I see you haven't merged PR yet, so go ahead and fix env before merge :)

Reply:
oops, good catch!

Reply:
I saw you fixed it so nice work :) 👍